iOS/CreditsViewController.swift

```diff
@@ -109,22 +109,19 @@ public class CreditsViewController: UIViewController, UIWebViewDelegate {
         doneButton.layer.cornerRadius = doneButton.frame.size.height / 2
         view.addSubview(doneButton)
 
-        // iPhoneX support
-        if #available(iOS 11.0, *) {
```

Comment:
I assume this availbility is going away because the minimum platform is iOS 11 now, It looks like there is one other one that is still there, should maybe remove that as well?

Reply:
👍

It looks like there is some inconsistency with the target and project ios deployment target settings so I'll try to fix that too.